docs/initial-milestones.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry: A Roadmap to Convergence
+
+This document covers the intial milestones for each SIG developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the goals and timelines for OpenTelemetry.
+
+If you are interested in starting a new SIG, please do the following:
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new SIG.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+## Initial Milestones
+For languages which have both an OpenTracing and OpenCensus implementation, we would like to achieve parity in OpenTelemetry by **September, 2019**, and sunset the existing OpenTracing and OpenCensus SIGs by **November, 2019**. 
+
+Parity can be defined as the following milestones:
+* A set of interfaces which define the OpenTelemetry Specification in a given programming language.
+* An SDK which can propagate context and record data in common formats.
```

Comment:
why we are not saying that SDK should satisfy specification? There will be a spec for SDK.

Reply:
I feel it is important to make it clear that the interfaces are separate from the SDK, so that it is possible to have more than one implementation. I would like suggestions on how to make requirements such as this clear to everyone.

Reply:
I just meant to keep it as separate line, bit instead of saying these two things - context and recording data - just say SDK implementing the specification

Reply:
It's a good question! I expect that there are features which we will want to implement in the SDK, but which are not strictly necessary for reaching parity and sunsetting the old projects. So we should to be specific about which SDK features are required to hit our deadlines, vs which could be added later without a problem.

I'm fine with reorganizing the SDK milestones, but that was my thinking.

Reply:
propagate context and record data in common format is very vague. Does it include metrics aggregation or it can be left out for instance? Maybe spec needs to have `1.0` requirements subfolder or something. Combined with the common test cases